src/Aether/Devices/Sensors/ObservableSgp4x.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Aether.CustomUnits;
+using Aether.Devices.Sensors.Metadata;
+using System;
+using System.Collections.Generic;
+using System.Device.I2c;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using UnitsNet;
+
+namespace Aether.Devices.Sensors
+{
+    internal class ObservableSgp4x : ObservableSensor, IObservableI2cSensorFactory
+    {
+        private readonly Drivers.Sgp4x _sensor;
+        private static readonly SensorDependency TemperatureDependency = new SensorDependency(Measure.Temperature, true);
+        private static readonly SensorDependency HumidityDependency = new SensorDependency(Measure.Humidity, true);
```

Comment:
Prefer to explicitly name params that aren't obvious.

Also, I think these don't need to be rquired? Isn't there a default if not given?

```suggestion
        private static readonly SensorDependency TemperatureDependency = new SensorDependency(Measure.Temperature, required: false);
        private static readonly SensorDependency HumidityDependency = new SensorDependency(Measure.Humidity, required: false);
```